lib/twilio-ruby/rest/task_router_client.rb

```diff
@@ -0,0 +1,203 @@
+require 'twilio-ruby/rest/base_client'
+module Twilio
+  module REST
+    class TaskRouterClient < BaseClient
+      API_VERSION = 'v1'
+
+      DEFAULTS = {
+          host: 'taskrouter.twilio.com',
+          port: 443,
+          use_ssl: true,
+          ssl_verify_peer: true,
+          ssl_ca_file: File.dirname(__FILE__) + '/../../../conf/cacert.pem',
+          timeout: 30,
```

Comment:
prolly want to set this to 31 to avoid contention between the client timeout and the timeout set on the load balancer